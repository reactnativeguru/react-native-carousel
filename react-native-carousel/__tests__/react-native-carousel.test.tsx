/**
 * @jest-environment jsdom
 */

import React from 'react'
import { shallow, mount } from 'enzyme'
import { RNCarousel } from '../dist'

const three = () => 3

const setup = (props = {sources: []}) => {
  const wrapper = shallow(<RNCarousel {...props} />)
  return wrapper
}

describe('RNCarousel renders correctly.', () => {
  let shallowWrapper
  it('RNCarousel container view renders correctly.', () => {
    shallowWrapper = setup()
    const rnCarousel = shallowWrapper.find({ testID: 'RNCarousel' })

    expect(rnCarousel.html()).toMatchSnapshot()
  })
})

test('3 is equal to 3', () => {
  expect(three()).toBe(3);
})
