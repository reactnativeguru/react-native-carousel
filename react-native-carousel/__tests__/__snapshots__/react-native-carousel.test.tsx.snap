// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RNCarousel renders correclty. OTP container view renders correctly. 1`] = `"<View testID=\\"RNCarousel\\" style=\\"flex:1;justify-content:center;align-items:center\\"></View>"`;
